### ISSUE TITLE
stager plugin execute tasks in parallel

### ISSUE THREAD

**Issue by romain-grecourt** [helidon-io/helidon-build-tools#743]
Sequential execution of tasks makes staging a lengthy process.
Implement a solution to parallelize the tasks execution in order to speed-up the staging process.

E.g. create a new task parallel that would be a wrapper for nested tasks to be executed in parallel.
